frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -87,6 +92,26 @@ export const EditZing = () => {
     }
   }
 
+  //Map for getting the names of templates based on ID for rendering tooltips
+  const [templateNameMap, setTemplateNameMap] = useState<{
+    [key: string]: string
+  }>({})
```

Comment:
This should be simpler once we centralize the templates across the app state later